.prettierrc.yml

```diff
@@ -0,0 +1,4 @@
+overrides:
+  - files: "*.md"
+    options:
+      proseWrap: always
```

Comment:
Are we sure we want this? It frequently mangles quote blocks and blocks of links. 

Have you had experiences with this?

Reply:
My experience has been pretty good with wrapping enabled, but it requires marking code blocks / quote blocks correctly :(

I'll change it to run only on changed files, and only on PRs (so we don't get a surprise on master).